react/features/video-layout/functions.js

```diff
@@ -15,7 +15,8 @@ import {
 import {
     getNumberOfPartipantsForTileView,
     getThumbnailMinHeight,
-    getTileDefaultAspectRatio
+    getTileDefaultAspectRatio,
+    shouldDisplayStageFilmstrip
```

Comment:
It seems I introduced this import from 'functions.web' and now I just fixed it in master by moving some of the functions. Potentially you may get some conflicts :(